app/dashboard/package.json

```diff
@@ -15,16 +15,17 @@
   },
   "scripts": {
     "compile": "tsc",
-    "typecheck": "tsc --noEmit",
+    "typecheck": "tsc",
```

Comment:
one of `compile` or `typecheck` are redundant now, right?

Reply:
It looks like that, but that's not really true. I wanted `tsc` to run here for both repo-wide `typecheck` and `compile` commands, since both of those unfortunately require the dashboard to emit its typings. The good thing is that it takes like a second to complete, so not a big deal.